src/main/java/sirius/kernel/health/Exceptions.java

```diff
@@ -490,4 +492,43 @@ public static Throwable getRootCause(@Nullable Throwable e) {
 
         return cause;
     }
+
+    /**
+     * Generates a stack trace for the given throwable and its causes without the error messages.
+     *
+     * @param throwable the throwable to generate the stack trace for
+     * @return a string representation of the stack trace without the error message
+     */
+    public static String printStackTraceWithoutErrorMessage(Throwable throwable) {
+        StringBuilder stringBuilder = new StringBuilder();
+
+        stringBuilder.append(throwable.getClass().getName()).append(":").append("\n");
+        appendStackTrace(stringBuilder, throwable);
+
+        try {
+            // The first element of the causal chain is always the throwable followed by its cause hierarchy.
+            // Therefore, the first element is skipped.
+            Throwables.getCausalChain(throwable).stream().skip(1).forEach(cause -> {
+                stringBuilder.append("Caused by: ").append(cause.getClass().getName()).append(":").append("\n");
+                appendStackTrace(stringBuilder, cause);
+            });
+        } catch (IllegalArgumentException exception) {
+            // This happens if the causal chain has a circular reference.
+            stringBuilder.append("Warning: Circular reference detected in causal chain. Skipping causes: ")
+                         .append(exception.getMessage());
+        }
+
+        return stringBuilder.toString();
+    }
+
+    /**
+     * Appends the stack trace of the given throwable to the given string builder.
+     *
+     * @param stringBuilder the string builder to append the stack trace to
+     * @param throwable     the throwable to append the stack trace of
+     */
+    private static void appendStackTrace(StringBuilder stringBuilder, Throwable throwable) {
+        Arrays.stream(throwable.getStackTrace())
+              .forEach(stackTraceElement -> stringBuilder.append("\t").append("at  ").append(stackTraceElement).append("\n"));
```

Comment:
Why a double space after `at`?

Reply:
Sorry that may have been my mistake in the suggestion i made